apple/internal/testing/apple_test_rule_support.bzl

```diff
@@ -135,6 +141,11 @@ def _apple_test_rule_impl(ctx, test_type):
     test_bundle_target = ctx.attr.deps[0]
 
     test_bundle = test_bundle_target[AppleTestInfo].test_bundle
+    swift_module_list = test_bundle_target[AppleTestInfo].swift_modules.to_list()
+
+    test_swift_module_file = swift_module_list[len(swift_module_list) - 1]
```

Comment:
Should this do a lenght check - if there is less than 1 swiftmodule it's out of bounds?